File: p.go
package main

func isPrime(number int) int {
	for a := 2; a < number; a++ {
      if number % a == 0 {
      	return 0
      }
   	}
   	return 1
}

func main() {
	count:=0
	// start := time.Now()
	for a := 2; a < 2000; a++ {
      if isPrime(a) == 1 {
      	count++;
      	// fmt.Print("X")
      }
   	}

	// fmt.Println("\n",count,"\nThe time is", time.Since(start))
}

File: p.php
<?php
	function isPrime($Number){
		for($i=2;$i<$Number;$i++){
			if ($Number%$i ==0)
				return false;
		}
		return true;
	}

	$count=0;
	// $time_start = microtime(true);
	for ($i=2; $i < 2000 ; $i++) { 
		if (isPrime($i)){
			$count++;
			// echo "X";
		}
	}
	// echo "\n";
	// echo $count;
    // $time_end = microtime(true);
    // $time = ($time_end - $time_start)*1000;
    // echo "\nProcess Time: {$time}\n";
?>